src/Symfony/Component/ErrorHandler/composer.json

```diff
@@ -27,6 +27,7 @@
         "symfony/http-kernel": "^3.4|^4.0|^5.0"
     },
     "autoload": {
+        "files": [ "Debug.php" ],
```

Comment:
should use class map

Reply:
I tried it, but it doesn't work properly with `classmap`, we need to autoload this file at startup so the alias is available in the `console.php` and `index.php` files, otherwise it breaks "Class not found".

Reply:
Any clue how to achieve it without adding it to autoload-dev files and without do this https://github.com/symfony/recipes/pull/630 neither?

Reply:
Any hint on how to get it without adding it to the autoload-dev files and without doing this either symfony/recipes#630?

Reply:
I don't get why classmap doesn't work - can we investigate?

Reply:
As far as I can see the `vendor/composer/autoload_real.php`, it loads the `autoload_files.php` file and automatically loads the defined files (`require $file`) when calling the `getLoader()` method. The others are lazy.

Reply:
It doesn't work because classes defined in the classmap are still loaded on demand. What we want here is to define the alias before any `Debug\Debug` usage. Putting `ErrorHandler\Debug` in `files` requires the file immediately, thus applying the alias before any usage possible.

However, the alias could be put in a separate file to reduce the load to what we exactly need.

Reply:
And this is what we want to achieve here:
```php
use Symfony\Component\Debug\Debug;

require 'vendor/autoload.php';

(new Debug())->enable();
```
It would make it possible to upgrade to 4.4 without changing the entry files (`console.php` and `index.php`).

Reply:
> However, the alias could be put in a separate file to reduce the load to what we exactly need.

What will be the load reduction in that case?

Reply:
Not having to load the `ErrorHandler\Debug` class if it's not used. But that's a supposition because I don't know if `class_alias` loads the original class or not.

Reply:
Yes, it does.

Reply:
which would fail.

Reply:
so this always loads the Debug class, even in prod mode now. not sure if we should sacrifice performance for an easier upgrade. that is why I suggested to keep the Debug class in the debug component.